helix-term/src/keymap.rs

```diff
@@ -360,7 +360,10 @@ impl Keymaps {
     pub fn get(&mut self, mode: Mode, key: KeyEvent) -> KeymapResult {
         // TODO: remove the sticky part and look up manually
         let keymaps = &*self.map();
-        let keymap = &keymaps[&mode];
+        let keymap = match keymaps.get(&mode) {
+            Some(keymap) => keymap,
+            None => return KeymapResult::NotFound,
+        };
```

Comment:
```suggestion
        let Some(keymap) = keymaps.get(&mode) else { return KeymapResult::NotFound; };
```
This is the style used elsewhere in the code.